Tests/DependencyInjection/Compiler/FormatListenerRulesPassTest.php

```diff
@@ -26,7 +26,7 @@ public function testRulesAreAddedWhenFormatListenerAndProfilerToolbarAreEnabled(
             ->setMethods(['hasDefinition', 'getDefinition', 'hasParameter', 'getParameter'])
             ->getMock();
 
-        $container->expects($this->exactly(3))
+        $container->expects($this->exactly(5))
```

Comment:
Actually, it does not really matter how often this stub is called an we could omit the `expects()` call entirely.